docs/config.rst

```diff
@@ -11,7 +11,7 @@ An example of a ``gordon-janitor.toml`` file:
     :language: ini
 
 
-You may choose to have a ``gordon-janitor-user.toml`` file for development. Any top-level key will override what's found in ``gordon-janitor.toml``.
+You may choose to have a ``gordon-janitor-user.toml`` file for development. All tables are deep merged into ``gordon-janitor.toml``, to limit the amount of config duplication needed. For example, you can override ``core.debug`` without having to redeclare which plugins you'd like to run).
```

Comment:
Maybe sections instead of tables?  You also now have an extra ).  I'm ok with it otherwise.

Reply:
Good catch. Tables is the language that toml uses. https://github.com/toml-lang/toml

Reply:
I think the question is if Jane User is going to check the official docs or just copy and modify the examples.  I want to be clear in both cases.  'tables (sections)'?

Reply:
I get that, but I think it's sufficient, given the example toml below where you can see that "core" is a section/table.

Reply:
Fair enough.